inference-engine/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/squared_difference.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <cstddef>
+#include <map>
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/squared_difference.hpp"
+
+using LayerTestsDefinitions::SquaredDifferenceLayerTest;
+
+namespace {
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+};
+
+const std::vector<std::vector<std::size_t>> inputShapes = {
+    {1, 30}, {1, 30}
```

Comment:
Maybe add a dimension to one of the shapes so we can confirm broadcast behavior is correct?